core/server/api_container/server/startosis_engine/plan_yaml/plan_yaml_generator.go

```diff
@@ -375,6 +397,20 @@ func (planYaml *PlanYaml) RemoveService(serviceName string) {
 	}
 }
 
+func (planYaml *PlanYamlGenerator) AddPackageDependencies(packageDependency map[string]bool) {
+	for dependency := range packageDependency {
+		planYaml.privatePlanYaml.PackageDependencies = append(planYaml.privatePlanYaml.PackageDependencies, dependency)
+	}
+	slices.Sort(planYaml.privatePlanYaml.PackageDependencies)
```

Comment:
This opens us up to duplicates, and as an engineer I'd expect `AddPackageDepencies` with a map-set argument to be idempotent. Would it be better to store the package dependencies as a map-set in the generator too, and only write it out to list at generation time?